app/Actions/Fortify/UpdateUserProfileInformation.php

```diff
@@ -21,12 +22,18 @@ public function update(User $user, array $input): void
             'name' => ['required', 'string', 'max:255'],
             'email' => ['required', 'email', 'max:255', Rule::unique('users')->ignore($user->id)],
             'photo' => ['nullable', 'mimes:jpg,jpeg,png', 'max:1024'],
+            'cv' => ['nullable', 'mimes:pdf', 'max:4096'],
         ])->validateWithBag('updateProfileInformation');
 
         if (isset($input['photo'])) {
             $user->updateProfilePhoto($input['photo']);
         }
 
+        if (isset($input['cv'])) {
+            $student = Student::where('user_id', $user->id)->first();
```

Comment:
```suggestion
            $student = Student::find($user->id);
```

This uses the primary key directly, whatever it may be. I believe it's a better way of doing this but feel free to ignore.